### PR TITLE
Executable tutorial: Learn to integrate Jest in your React-Redux projects

### PR DESCRIPTION
**Pull request by ghost**
# Proposal
A locally hosted tutorial with the goal to teach the user how to set up a test-environment for React-Redux projects using Jest, and integrating it with the GitHub actions pipeline.

The user will learn how to write unit and integration tests, integrate them with a GitHub repository, and get a better understanding of why it is important and relevant to have good test coverage.

## Members
- Agnes Petäjävaara <agnespet@kth.se> anorangesky(https://github.com/anorangesky)
- Adam Henriksson <adahen@kth.se> adamih(https://github.com/Adamih)

### CODE REVIEW

File: contributions/executable-tutorial/agnespet-adahen/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Proposal
+A locally hosted tutorial with the goal to teach the user how to set up a test-environment for React-Redux projects using Jest, and integrating it with the github actions pipeline.
```
Comment (cesarsotovalero) [inline contributions/executable-tutorial/agnespet-adahen/README.md]: What do you mean with "locally hosted tutorial"?

Comment (cesarsotovalero) [inline contributions/executable-tutorial/agnespet-adahen/README.md]: What is Jest? Can you add more information?

**Review by cesarsotovalero — CHANGES_REQUESTED** [review]
Please provide more details about your proposal.

File: contributions/executable-tutorial/agnespet-adahen/README.md (hunk 1 above)
Comment (ghost) [reply contributions/executable-tutorial/agnespet-adahen/README.md]: I've updated the PR

**Review by cesarsotovalero — APPROVED** [review]

**Comment by andnil5** [comment]
We plan to give feedback on this tutorial.

**Comment by ghost** [comment]
The proposal has changed, the new PR is here #1242

**Pull request merged**